### PR TITLE
my fat review of prs we merged yesterday

### PR DESCRIPTION
**Pull request by Ktlwjec1**
ok here so no one is confused:

- code/modules/research/designs/AI_module_designs.dm + code/datums/ai_laws.dm + code/datums/diseases/advance/symptoms/necropolis.dm added full stops (.) to some sentences
- code/modules/food_and_drinks/recipes/tablecraft/recipes_sandwich.dm contains /datum/crafting_recipe/food/icecreamsandwich which is duplicated in code/modules/food_and_drinks/recipes/tablecraft/recipes_frozen.dm so I removed it
- code/modules/food_and_drinks/food/snacks_sandwichtoast.dm contains /obj/item/reagent_containers/food/snacks/icecreamsandwich which is duplicated in code/modules/food_and_drinks/food/snacks_frozen.dm so I removed it
- yogstation/icons/obj/food/food.dmi contains the icecream sandwich icon which is duplicated in icons/obj/food/food.dmi so I removed it
- code/modules/food_and_drinks/recipes/tablecraft/recipes_misc.dm contains /datum/crafting_recipe/food/spacefreezy, /datum/crafting_recipe/food/sundae and /datum/crafting_recipe/food/honkdae which are duplicated in code/modules/food_and_drinks/recipes/tablecraft/recipes_frozen.dm so I removed them
- code/modules/food_and_drinks/food/snacks_frozen.dm spelling errors fixed in this
- switched laughter + pwr game around from clown + pwr game snowcones
- changed honey_sc to lemon_sc
- code/modules/food_and_drinks/recipes/tablecraft/recipes_frozen.dm spelling errors and /datum/crafting_recipe/food/honey_sc changed to /rainbow_sc as there was 2 honey ones
- icons: removed unused kiwi icon, duplicated icon removed (honey_sc), mime_sc had a black dot I've removed

### CODE REVIEW

**Review by alexkar598 — CHANGES_REQUESTED** [review]
You reverted a PR...

**Comment by Ktlwjec1** [comment]
no?

**Comment by patpol4** [comment]
dizzled

File: yogstation.dme [hunk 1]
```diff
@@ -1599,6 +1599,7 @@
 #include "code\modules\clothing\gloves\_gloves.dm"
 #include "code\modules\clothing\gloves\boxing.dm"
 #include "code\modules\clothing\gloves\color.dm"
+#include "code\modules\clothing\gloves\knifeboxing.dm"
```
Comment (JamieD1) [inline yogstation.dme:1602]: Can you just discard the changes on this file?

**Review by JamieD1 — CHANGES_REQUESTED** [review]

File: yogstation.dme (hunk 1 above)
Comment (Ktlwjec1) [reply yogstation.dme:1602]: It makes it alphabetical, like it should be.

**Review by alexkar598 — APPROVED** [review]

**Pull request merged**